addons/binding/org.openhab.binding.sensebox/org.openhab.binding.sensebox.iml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes, adding *.iml (and maybe .idea) to the .gitignore at the root would be fine. Until then, I'll remove this file from my commit, too.